src/AppBundle/Service/OrderManager.php

```diff
@@ -194,6 +187,39 @@ public function capturePayment(OrderInterface $order)
             $stripePayment->setLastError($e->getMessage());
             $stateMachine->apply(PaymentTransitions::TRANSITION_FAIL);
```

Comment:
In my understanding it won't cause money issue because the Transfer is in the same `transfer_group` as the Charge, so it won't cause issue if the charge fails.

Nonetheless I added a return statement in ae138040